src/less/nav.less

```diff
@@ -245,3 +236,25 @@
 }
 
 }
+
+global-menu-container {
```

Comment:
@pascualin can you investigate why it's displaying on all.html? also, the position appears to be broken on each of the example pages. the backdrop isn't fixed and it's scrolling with the underlying page content.
![image](https://user-images.githubusercontent.com/130654/26942169-e4826dd4-4c35-11e7-97c7-3291525fe9ff.png)